src/modm/ui/display/graphic_display.hpp

```diff
@@ -19,461 +20,437 @@
 #define MODM_GRAPHIC_DISPLAY_HPP
 
 #include <modm/architecture/interface/accessor.hpp>
-#include <modm/math/geometry.hpp>
-
 #include <modm/io/iodevice.hpp>
 #include <modm/io/iostream.hpp>
+#include <modm/math/geometry.hpp>
 
 #include "font.hpp"
 
 namespace modm
 {
-	/// @ingroup	modm_ui_display
-	namespace glcd
+/// @ingroup	modm_ui_display
+namespace glcd
+{
+/// @ingroup modm_ui_display
+using Point = Vector<int16_t, 2>;
+
```

Comment:
```suggestion

/// @ingroup modm_ui_display
```

Unfortunately Doxypress is a little annoying.